sequence_task_1/student/include/impl.h

```diff
@@ -107,16 +99,15 @@ class StaticPlaylist
     /** Get first track in playlist stack */
     const Song& current() const
     {
-        return m_tracklist.front();
+        return m_tracklist.back();
     }
 
     /** Skip to the next track in playlist, remove current */
     void switchNext()
     {
         if(hasTracks())
         {
-            //first is current, so erase first, second will be current
-            m_tracklist.erase(m_tracklist.begin());
+            (void)m_tracklist.pop_back();
```

Comment:
why did you casted it to void?

Reply:
shown that i ignore return value, code stile in my project

Reply:
Looks like a bug, pop_back returns nothing